core/src/main/java/io/grpc/internal/AbstractServerStream.java

```diff
@@ -168,6 +168,11 @@ public final boolean isReady() {
   }
 
   @Override
+  public String getAuthority() {
+    return "";
```

Comment:
Given that `HandlerRegistry` is the only thing that is using the value from `getAuthority()` currently, and it is `@Nullable`, I would still feel better by defaulting to `null`. Here and in `NettyServerHandler`.

It's something we can change in the future though. We'll probably deal with it when we expose `getAuthority()` on `ServerCall`. At that point we may make it non-`null` where empty string is explicitly called out in the API.